test/unit_tests/braket/pulse/test_pulse_sequence.py

```diff
@@ -337,3 +337,86 @@ def test_pulse_sequence_to_ir(predefined_frame_1, predefined_frame_2):
         ]
     )
     assert pulse_sequence.to_ir() == expected_str
+
+
+def test(predefined_frame_1, predefined_frame_2):
```

Comment:
Please update the test name to be something more descriptive

Reply:
Fixed